lib/src/templates/big_fact_slide.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_deck/src/flutter_deck_layout.dart';
+import 'package:flutter_deck/src/templates/templates.dart';
+
+/// A slide widget that represents a big fact Slide
+/// This class is used to create the big fact slide in a slide deck.
+/// It renders two lines of text, a Big title and a subtitle.
+/// It reders the default header and footer of the slide deck, if they are
+/// enabled in the configuration.
+
+class FlutterDeckBigFactSlide extends StatelessWidget {
+  /// Creates a new title slide.
+  /// The [title] and [subtitle] arguments must not be null.
+  /// The [backgroundBuilder] argument is optional.
+  /// The [titleMaxLines] is the maximum number of lines for the title by
+  /// default = 3.
+  const FlutterDeckBigFactSlide({
+    required this.title,
+    this.subtitle,
+    this.titleMaxLines,
+    this.backgroundBuilder,
+    super.key,
+  });
+
+  /// The title of the slide.
+  final String title;
+
+  /// The subtitle of the slide.
+  final String? subtitle;
+
+  /// maxLines for the title of the slide.
+  final int? titleMaxLines;
```

Comment:
I am not so sure about this property. I think the title should always fit a single line - it's a short number or fact. However, I agree that we can limit the subtitle to 3 lines. Thus, rename this property to `subtitleMaxLines` and move the logic to the subtitle.

Reply:
Yes makes sense, so now I should just add 3 max lines by default for the subtitle, and remove it from title ?